README.md

```diff
@@ -238,128 +264,131 @@ Invoke-WebRequest -Uri https://github.com/docker/app/releases/download/v0.6.0/do
+`docker-compose.yml` and `parameters.yml`.
 
-Converting between the two formats can be achieved by using the `docker-app split` and `docker-app merge` commands.
+Converting between the two formats can be achieved by using the
+`docker app split` and `docker-app merge` commands.
 
-Note that you cannot store attachments in the single file format. If you want to use attachments you should use the directory format.
+**Note**: You cannot store attachments in the single file format. If you want to
+use attachments you should use the directory format.
 
 ## Attachments (Storing additional files)
 
-If you want to store additional files in the application package, such as `prod.yml`, `test.yml` or other config files, use the directory format and simply place these files inside the *.dockerapp/ directory. These will be bundled into the package when using `docker-app push`.
+If you want to store additional files in the application package, such as
+`prod.yml`, `test.yml` or other config files, use the directory format and
+simply place these files inside the *.dockerapp/ directory. These will be
+bundled into the package when using `docker app push`.
 
 ## Sharing your application on the Hub
 
-You can push any application to the Hub using `docker-app push`:
+You can push any application to the Hub using `docker app push`:
 
-``` bash
-$ docker-app push --tag myhubuser/myimage:latest
+```console
+$ docker app push --tag myhubuser/myimage:latest
 ```
 
 This command will push to the Hub an image named `myhubuser/myimage:latest`.
 
-If you omit the `--tag myhubuser/myimage:latest` argument, this command uses the application `version` defined in `metadata.yml` as the tag.
+If you omit the `--tag myhubuser/myimage:latest` argument, this command uses the
+application `version` defined in `metadata.yml` as the tag.
 
-All `docker-app` commands accept an image name as input, which means you can run on a different host:
+All `docker app` commands accept an image name as input, which means you can run
+on a different host:
 
-``` bash
-$ docker-app inspect myhubuser/myimage
+```console
+$ docker app inspect myhubuser/myimage
 ```
 
 ## Next steps
 
-We have lots of ideas for making Compose-based applications easier to share and reuse, and making applications a first-class part of the Docker toolchain. Please let us know what you think about this initial release and about any of the ideas below:
+We have lots of ideas for making Compose-based applications easier to share and
+reuse, and making applications a first-class part of the Docker toolchain.
+Please let us know what you think about this initial release and about any of
+the ideas below:
 
 * Introducing environments to the parameters file
 * Docker images which launch the application when run
 * Built-in commands for running applications
 * Saving required images into the application artifact to support offline installation
 * Signing applications with notary
```

Comment:
Are those topics still up to date ?

Reply:
Nope. I'll remove them until we have a more clear roadmap to share.